scripts/wait-for-cluster.sh

```diff
@@ -13,13 +13,21 @@
 # See the License for the specific language governing permissions and
```

Comment:
Back this into a new PR.

Reply:
See https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/46